src/init/init.c

```diff
@@ -31,6 +31,13 @@
 #if CONFIG_IPC_MAJOR_4
 #include <ipc4/fw_reg.h>
 #endif
+#include <user/abi_dbg.h>
```

Comment:
we need this (and the rest) without ZEPHYR_LOG?

Reply:
Ack @abonislawski , will fix in V3.